### DISCUSSION TITLE
article/zhou-zuo-luo-write-lesson

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#95] [Announcements]
# article/zhou-zuo-luo-write-lesson

写作是倒逼成长的绝佳办法：如果你今天没有学习，没有成长，没有对一件事进行深入的思考，你就不知道写什么；如果你每天都有东西写，而且写出来还不错，说明今天的你和昨天的你相比，又有了新的进步。

https://blog.tangly1024.com/article/zhou-zuo-luo-write-lesson

**Comment by xianyu110** [comment]
awesome
